index.bs

```diff
@@ -848,6 +848,36 @@ try to keep its syntax as close as possible to the syntax of the existing attrib
 	and the other one a selector, that would be an antipattern.
 </div>
 
+## Do not pause the HTML parser
+
+Ensure that your design does not require HTML parser to pause to handle external resources.
+
+As a page parses,
+the browser discovers assets that the page needs,
+and figures out a priority in which they should be loaded in parallel.
+This is disrupted by a resource which blocks the discovery of subsequent resources.
+At worst, it means the browser downloads items in series rather than parallel.
+At best, it means the browser queues downloads based on speculative parsing,
+which may turn out to be incorrect.
+
+Features that block the parser generally do so
+because they want to feed additional content into the parser
+before subsequent content.
+This is the case of legacy `<script src="…">` elements,
+which can inject into the parser using `document.write(…)`.
+Due to the performance issues above, new features must not do this.
+
+## Be cautious about blocking the renderer
+
+Block rendering only if it improves user experience.
```

Comment:
```suggestion
## Avoid features that block rendering

Features that block rendering should be considered only if user experience is improved.
```

Reply:
"Avoid pausing the HTML parser"
"Avoid features that block rendering"

At a glance, the wording here suggests both rules carry equal weight, but pausing the parser is much worse than blocking rendering.

Reply:
That's fair. Render blocking is certainly a subset of parser blocking in terms of badness. Render blocking is also more combinatorial compared to blocking the HTML parser.

I'll revert the HTML parser section heading so it holds stronger.